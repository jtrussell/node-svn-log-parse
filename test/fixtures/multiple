------------------------------------------------------------------------
r491 | bruce_lee | 2013-10-11 15:03:08 +0000 (Fri, 11 Oct 2013) | 4 lines

test: Validate black belt is not really shade of gray

Appears to be legit

------------------------------------------------------------------------
r490 | chuck_norris | 2013-08-29 15:29:07 +0000 (Thu, 29 Aug 2013) | 1 line

feature: Add roundhouse kick
------------------------------------------------------------------------
r489 | bruce_lee | 2013-08-22 19:35:20 +0000 (Thu, 22 Aug 2013) | 1 line

fix: Typo
------------------------------------------------------------------------