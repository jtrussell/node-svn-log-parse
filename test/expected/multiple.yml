---
- revision: 'r491'
  committer: 'bruce_lee'
  time: '2013-10-11 15:03:08 +0000 (Fri, 11 Oct 2013)'
  line_count: '2 lines'
  message: |
            test: Validate black belt is not really shade of gray
            
            Appears to be legit
            
- revision: 'r490'
  committer: 'chuck_norris'
  time: '2013-08-29 15:29:07 +0000 (Thu, 29 Aug 2013)'
  line_count: '1 line'
  message: 'feature: Add roundhouse kick'
  
- revision: 'r489'
  committer: 'bruce_lee'
  time: '2013-08-22 19:35:20 +0000 (Thu, 22 Aug 2013)'
  line_count: '2 lines'
  message: 'fix: Typo'